cronjobs/group-sync/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - clusterrole.yaml
+  - clusterrolebinding.yaml
+  - cronjob.yaml
+  - rolebinding.yaml
+  - serviceaccount.yaml
+namespace: cs210-computer-system-boston-university-e5839d
```

Comment:
Can we sync 2 class groups in one cronjob?

Reply:
That's technically possible, though would complicate the code a bit. I'm not too worried about deploying a script per namespace since this is basically set and forget and only needs to be created once per namespace. @DanNiESh what do you think?
I will update the namespace to require manual update in kustomization.yaml though

Reply:
Cool.